purchase_discount/__manifest__.py

```diff
@@ -4,20 +4,22 @@
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
-    "name": "Purchase order lines with discounts",
+    "name": "Purchase Order Lines with Discounts",
```

Comment:
Why making this unneeded changes?

Reply:
I thought the module name should be in upper case letters.
@pedrobaeza

Reply:
I don't think so. Please respect it as it was.